test/bats/aws.bats

```diff
@@ -0,0 +1,126 @@
+    wait_for_process $WAIT_TIME $SLEEP_TIME "$cmd"
+
+    run kubectl get crd/secretproviderclasses.secrets-store.csi.x-k8s.io
+    assert_success
+}
+
+@test "deploy aws secretproviderclass crd" {
+    envsubst < $BATS_TEST_DIR/BasicTestMountSPC.yaml | kubectl --namespace $NAMESPACE apply -f -
+
+    cmd="kubectl --namespace $NAMESPACE get secretproviderclasses.secrets-store.csi.x-k8s.io/basic-test-mount-spc -o yaml | grep aws"
+    wait_for_process $WAIT_TIME $SLEEP_TIME "$cmd"
+}
+
+@test "CSI inline volume test with pod portability" {
+   kubectl --namespace $NAMESPACE  apply -f $BATS_TEST_DIR/BasicTestMount.yaml
+   cmd="kubectl --namespace $NAMESPACE  wait --for=condition=Ready --timeout=60s pod/basic-test-mount"
+   wait_for_process $WAIT_TIME $SLEEP_TIME "$cmd"
+
+   run kubectl --namespace $NAMESPACE  get pod/$POD_NAME
+   assert_success
+}
+
+@test "CSI inline volume test with rotation - parameter store " {
+   result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/ParameterStoreRotationTest)
+   [[ "${result//$'\r'}" == "BeforeRotation" ]]
+
+   aws ssm put-parameter --name ParameterStoreRotationTest --value AfterRotation --type SecureString --overwrite --region $REGION
+   sleep 40
+   result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/ParameterStoreRotationTest)
+   [[ "${result//$'\r'}" == "AfterRotation" ]]
+}
+
+@test "CSI inline volume test with rotation - secrets manager " {
+   result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/SecretsManagerRotationTest)
+   [[ "${result//$'\r'}" == "BeforeRotation" ]]
+  
+   aws secretsmanager put-secret-value --secret-id SecretsManagerRotationTest --secret-string AfterRotation --region $REGION
+   sleep 40
+   result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/SecretsManagerRotationTest)
+   [[ "${result//$'\r'}" == "AfterRotation" ]]
+}
+
+@test "CSI inline volume test with pod portability - read ssm parameters from pod" {
+   result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/ParameterStoreTest1)
+   [[ "${result//$'\r'}" == "ParameterStoreTest1Value" ]]
+
+   result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/ParameterStoreTest2)
+   [[ "${result//$'\r'}" == "ParameterStoreTest2Value" ]]
+}
+
+@test "CSI inline volume test with pod portability - read secrets manager secrets from pod" {
+    result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/SecretsManagerTest1)
+    [[ "${result//$'\r'}" == "SecretsManagerTest1Value" ]]
+   
+    result=$(kubectl --namespace $NAMESPACE exec $POD_NAME -- cat /mnt/secrets-store/SecretsManagerTest2)
+    [[ "${result//$'\r'}" == "SecretsManagerTest2Value" ]]        
+}
+
+@test "Sync with Kubernetes Secret" { 
+    run kubectl get secret --namespace $NAMESPACE  secret
+    [ "$status" -eq 0 ]
```

Comment:
`assert_success` will check `status -eq 0`

Reply:
Will change to assert_success